manifests/l/LightBurnSoftware/LightBurn/1.3.00/LightBurnSoftware.LightBurn.locale.en-US.yaml

```diff
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS2.5-1-22610-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: LightBurnSoftware.LightBurn
+PackageVersion: 1.3.00
+PackageLocale: en-US
+Publisher: LightBurn Software
+PublisherUrl: https://lightburnsoftware.com/
+PublisherSupportUrl: https://forum.lightburnsoftware.com/
+# PrivacyUrl: 
+# Author: 
+PackageName: LightBurn
+PackageUrl: https://lightburnsoftware.com/
+License: Proprietary
+# LicenseUrl: 
+Copyright: Created by LightBurn Software, LLC
```

Comment:
```suggestion
Copyright: © 2022, LightBurn Software
```

Reply:
This does not align with what is in the Help->About window.

<img width="275" alt="image" src="https://user-images.githubusercontent.com/54531/206872898-49cd5efd-7cd0-42af-8489-0b0f22fd9d2c.png">